CMakeLists.txt

```diff
@@ -217,6 +217,12 @@ if(ASPECT_WITH_WORLD_BUILDER)
   ENDIF()
 ENDIF()
 
+# include Fastscape and Xtensor files 
+INCLUDE_DIRECTORIES("${CMAKE_SOURCE_DIR}/contrib/fastscape/include/")
+INCLUDE_DIRECTORIES("${CMAKE_SOURCE_DIR}/contrib/xtensor/include/")
+INCLUDE_DIRECTORIES("${CMAKE_SOURCE_DIR}/contrib/xtensor/include/xtensor/")
+INCLUDE_DIRECTORIES("${CMAKE_SOURCE_DIR}/contrib/xtl/include/")
```

Comment:
Hi @benbovy, thanks for your comments, it's true that it will be a better way of doing it, I'll give it a try, it also makes more sense for the license problem. This was a quick way to get started and will have to be modified.